redux-form/index.d.ts

```diff
@@ -1,476 +1,38 @@
-// Type definitions for redux-form v4.0.3
+// Type definitions for redux-form v6.3.1
 // Project: https://github.com/erikras/redux-form
-// Definitions by: Daniel Lytkin <https://github.com/aikoven>
+// Definitions by: Carson Full <https://github.com/carsonf>
```

Comment:
Add yourself onto the end. `// Definitions by: Daniel Lytkin <https://github.com/aikoven>, Carson Full <https://github.com/carsonf>`